src/app/dashboard/users/page.tsx

```diff
@@ -27,14 +32,27 @@ type UserRequestCounts =
     }
   | undefined
 
+async function getNonActiveUser(id: number): Promise<TautulliItemRow> {
+  const user = await fetchTautulli<TautulliItemRow>('get_user', {
+    user_id: id,
+  })
+  const nonActive = user?.response?.data ?? ({} as TautulliItemRow)
+
+  nonActive.total_duration = 0
```

Comment:
I think there is a misunderstanding. The user always exists, its just that he wasnt active in the defined time period. This means `total_duration` is undefined, resulting in an wrongly formatted badge in the ui. The null coalescing might be a bit misleading here. It is just here since the returned value is nullable. But this should never happen since we know the `user_id` exists, since the user is logged in...
```

const user = await fetchTautulli<TautulliItemRow>('get_user', {
    user_id: id,
  })
  const nonActive = user!.response.data

  nonActive.total_duration = 0

  return nonActive
}
```
I refactored the method to surpress the null warning

Reply:
`fetchTautulli` might still return `null` in case of an error.